ngraph/python/tests/test_frontend/test_frontend_onnx_editor.py

```diff
@@ -0,0 +1,551 @@
+    ]
+    output_tensors = [
+        make_tensor_value_info("out3", onnx.TensorProto.FLOAT, (2, 2)),
+        make_tensor_value_info("add_out", onnx.TensorProto.FLOAT, (2, 2)),
+    ]
+    graph = make_graph([add, relu], "test_graph", input_tensors, output_tensors)
+    models["test_override_all_outputs.onnx"] = make_model(graph, producer_name="ONNX Importer")
+
+    # Expected for test_override_all_outputs 2
+    input_tensors = [
+        make_tensor_value_info("in1", onnx.TensorProto.FLOAT, (2, 2)),
+        make_tensor_value_info("in2", onnx.TensorProto.FLOAT, (2, 2)),
+    ]
+    output_tensors = [
+        make_tensor_value_info("out4", onnx.TensorProto.FLOAT, (2, 2)),
+    ]
+    graph = make_graph([add, mul], "test_graph", input_tensors, output_tensors)
+    models["test_override_all_outputs_2.onnx"] = make_model(graph, producer_name="ONNX Importer")
+
+    # Expected for test_override_all_inputs
+    input_tensors = [
+        make_tensor_value_info("in3", onnx.TensorProto.FLOAT, (2, 2)),
+        make_tensor_value_info("out1/placeholder_port_0", onnx.TensorProto.FLOAT, (2, 2)),
+        make_tensor_value_info("out4/placeholder_port_0", onnx.TensorProto.FLOAT, (2, 2)),
+        make_tensor_value_info("out4/placeholder_port_1", onnx.TensorProto.FLOAT, (2, 2)),
+    ]
+    output_tensors = [
+        make_tensor_value_info("out1", onnx.TensorProto.FLOAT, (1, 2)),
+        make_tensor_value_info("out2", onnx.TensorProto.FLOAT, (1, 2)),
+        make_tensor_value_info("out3", onnx.TensorProto.FLOAT, (2, 2)),
+        make_tensor_value_info("out4", onnx.TensorProto.FLOAT, (2, 2)),
+    ]
+    expected_split = onnx.helper.make_node("Split", inputs=["out1/placeholder_port_0"],
+                                           outputs=["out1", "out2"])
+    expected_mul = onnx.helper.make_node("Mul", inputs=["out4/placeholder_port_0", "out4/placeholder_port_1"],
+                                         outputs=["out4"])
+    graph = make_graph([expected_split, relu, expected_mul], "test_graph", input_tensors, output_tensors)
+    models["test_override_all_inputs.onnx"] = make_model(graph, producer_name="ONNX Importer")
+
+    # test partial shape
+    input_tensors = [
+        make_tensor_value_info("in1", onnx.TensorProto.FLOAT, (8, 16)),
+        make_tensor_value_info("in2", onnx.TensorProto.FLOAT, (8, 16)),
+        make_tensor_value_info("in3", onnx.TensorProto.FLOAT, (4, 6)),
+    ]
+    output_tensors = [
+        make_tensor_value_info("out1", onnx.TensorProto.FLOAT, (4, 16)),
+        make_tensor_value_info("out2", onnx.TensorProto.FLOAT, (4, 16)),
+        make_tensor_value_info("out3", onnx.TensorProto.FLOAT, (4, 6)),
+        make_tensor_value_info("out4", onnx.TensorProto.FLOAT, (8, 16)),
+    ]
+    graph = make_graph([add, split, relu, mul], "test_graph", input_tensors, output_tensors)
+    models["test_partial_shape.onnx"] = make_model(graph, producer_name="ONNX Importer")
+
+    return models
+
+
+fem = FrontEndManager()
```

Comment:
Could you consider to reuse 'C++' tests for testing C++ frontends, like we did for 'PaddlePaddle'? There we generate necessary test models on build time, and then converting those models during testing? Starting from today, these tests are also executed on openvino-lin CI

Reply:
yes, it seems to be a good idea, but please let us do it separately. I created a ticket to handle it: 60998